src/sentry/sentry_metrics/indexer/mock.py

```diff
@@ -18,25 +19,43 @@
     "staging": 10,
     "user": 11,
     "init": 12,
+    "session.error": 13,
 }
```

Comment:
Nit. Why a dictionary here since all the indexes are sequential, a simple array would do (and just use the index in the array as the value).